src/app/sharedUI/molecules/ImageModal.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNativeZoomableView } from "@openspacelabs/react-native-zoomable-view"
+import { Image, Modal } from "react-native"
+import { useSafeAreaInsets } from "react-native-safe-area-context"
+import { BackButton, Flex } from "palette"
+
+interface ImageModalProps {
+  modalVisible: boolean
+  setModalVisible: (x: boolean) => void
+  uri: string
+}
+
+export const ImageModal = ({ modalVisible, setModalVisible, uri }: ImageModalProps) => {
+  const insets = useSafeAreaInsets()
+
+  return (
+    <Modal
+      visible={modalVisible}
+      onRequestClose={() => {
+        setModalVisible(false)
+      }}
+      animationType="fade"
+      presentationStyle="fullScreen"
+    >
+      <Flex backgroundColor="background" flex={1} pt={insets.top} px={2}>
```

Comment:
```suggestion
      <Flex backgroundColor="background" flex={1} pt={insets.top}>
```

i think this is not needed, so the image takes full width of screen?

Reply:
Addressed, resolving conversation 🙌🏻